src/stories/Button.stories.ts

```diff
@@ -0,0 +1,19 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { ButtonComponent, ButtonStyle } from '@/components/formElements';
+
+const meta = {
+  component: ButtonComponent
+} satisfies Meta<typeof ButtonComponent>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Example = {
+  args: {
+    label: 'Register',
+    style: ButtonStyle.DARK,
+    disabled: false,
+    loading: false
+  }
```

Comment:
add all args add them by exact them in button component
and also naming the component should be simple example
- if it's components folder then the files should be 
- Button not ButtonComponents

Reply:
Done 👍